ts/client/src/accounts/group.ts

```diff
@@ -466,6 +466,18 @@ export class Group {
     return banks[0];
   }
 
+  public getFirstBankForMngo(): Bank {
+    return this.getFirstBankByTokenIndex(this.mngoTokenIndex);
+  }
+
+  public getFirstBankForFees(): Bank {
+    return this.getFirstBankByTokenIndex(0);
+  }
+
+  public getFirstBankForSettlement(): Bank {
+    return this.getFirstBankByTokenIndex(0);
```

Comment:
Maybe be more specific and say "PerpSettlement"? Not a strong opinion though.

"ForFees" I don't get and seems strictly too generic. Fees can apply on any bank after all.